.github/workflows/main.yml

```diff
@@ -50,6 +50,34 @@ jobs:
           name: screenshots-${{ matrix.os }}-${{ matrix.version }}
           path: ${{ github.workspace }}/test/**/screenshots/*.png
 
+  publish-wiki:
+    runs-on: ubuntu-latest
+    name: Publish wiki
+    defaults:
+      run:
+        working-directory: ${{ github.workspace }}
+
+    env:
+      ORG: 'redhat-developer'
+      REPO_SOURCE: 'vscode-extension-tester'
+
+    steps:
+      - name: Checkout main repository
+        uses: actions/checkout@v4
+        with:
+          repository: ${{ format('{0}/{1}', env.ORG, env.REPO_SOURCE) }}
+          path: ${{ format('{0}/{1}', github.workspace, env.REPO_SOURCE) }}
+
+      - name: Upload Release Notes to Wiki
```

Comment:
I thinks something like `Publish Wiki pages` is more precise naming for this pipeline step.. WDYT?

Reply:
`ctrl + c` is the greatest invention of mankind. 
Will fix.